go/master/python/setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup, Distribution
```

Comment:
I am not familiar with Python packaging, can you let me know what's the better way to do it? I will make the change. Thanks!